README

```diff
@@ -1,3 +1,55 @@
-This is the unit-test for ACRN project. We have two type of unit-test:
- - guest side. The unit-test will be run as a ACRN guest
- - host side. The uni-test will be run at ACRN host side.
+# Readme
+  This is the unit-test for ACRN project. The unit-test will run as a ACRN guest.
+  Notes: acrn-unit-test now is still test framework ready only, no much detailed test case yet. 
+         But you can try and add new cases according to the following descriptions.
+
+# acrn-unit-test directory structure.
+    ./guest:			configure script, top-level Makefile
+    ./guest/scripts:		helper scripts for building and running tests
+    ./guest/lib:		general architecture neutral services for the tests
+    ./guest/lib/x86:		architecture dependent services for the tests
+    ./guest/x86:		the sources of the tests and the created objects/images
+
+# acrn-unit-test building steps.
+  1.Get unit-test code
+  $git clone https://github.com/projectacrn/acrn-unit-test.git  # branch: master
+
+  2:Enter acrn-unit-test/guest, config and compile. This directory contains sources for ACRN test suite.
+  To create all test images do:
+
+    ./configure --arch=x86_64
+    make
+
+  in this directory. Test images are created in ./x86/*.flat
+
+
+  or you can create one special test image:
+
+    ./configure --arch=x86_64
+    make ./x86/msr.flat
+
+  in this directory. Test image is created in ./x86/msr.flat
+  
+# ACRN Hypervisor setup steps, please refer to https://projectacrn.github.io/latest/getting-started/getting-started.html.
+  
+
+# Running acrn-unit-test based on ACRN shared scenario.
+  1.Boot ACRN and Service VM, SSH to Service VM.
+  
+  2.Copy *.flat to Service VM, and run acrn-unit-test.
+    $./dm.sh *.flat
+    
+    Example for dm.sh:
+    #!/bin/bash
+    /usr/bin/acrn-dm -A -m 1024M --debugexit -E $1 -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio -U 615db82a-e189-4b4f-8dbb-d321343e4ab3 acrn-vm
+
+  3.Check the test log in HV console.
+    ACRN:\>vm_console 1(VM ID of running *.flat)
```

Comment:
Just curious: does users still have to use the HV console to get the serial log from post-launched VMs?